src/gui/Src/BasicView/Disassembly.cpp

```diff
@@ -1928,7 +1928,7 @@ QString Disassembly::getAddrText(dsint cur_addr, char label[MAX_LABEL_SIZE])
         *label_ = 0;
     if(label)
         strcpy_s(label, MAX_LABEL_SIZE, label_);
-    return addrText;
+    return std::move(addrText);

```

Comment:
Shouldn't the compiler already be able to perform return value optimization here, since the function exclusively returns 'addrText'?  

In fact, I believe it can actually hurt performance (see [here](https://www.ibm.com/developerworks/community/blogs/5894415f-be62-4bc0-81c5-3956e82276f3/entry/RVO_V_S_std_move?lang=en) and [here](http://stackoverflow.com/questions/17473753/c11-return-value-optimization-or-move)).